activerecord/test/cases/quoting_test.rb

```diff
@@ -71,8 +71,19 @@ def test_quoted_time_utc
         with_timezone_config default: :utc do
           t = Time.now.change(usec: 0)
 
-          expected = t.getutc.change(year: 2000, month: 1, day: 1)
-          expected = expected.to_s(:db).sub("2000-01-01 ", "")
+          expected = t.change(year: 2000, month: 1, day: 1)
+          expected = expected.getutc.to_s(:db).sub("2000-01-01 ", "")
+
+          assert_equal expected, @quoter.quoted_time(t)
+        end
+      end
+
+      def test_quoted_time_dst
+        with_timezone_config default: :utc do
+          t = Time.now.change(usec: 0)
+
+          expected = t.change(year: 2000, month: 1, day: 1)
+          expected = expected.getutc.to_s(:db).sub("2000-01-01 ", "")
```

Comment:
Can you change this test to check that a time that is currently in DST is not in DST when quoted. You'll need to change `with_timezone_config` to use a zone other than `:utc` and use a fixed date/time rather than relying on `Time.now`. You'll also need to test this for when the Active Record timestamp format is both `:utc` and `:local`.